cypress/features/regression/experimental/dataInputClassic.feature

```diff
@@ -0,0 +1,120 @@
+Feature: Experimental Date Input classic component
+  I want to change Experimental Date Input classic component properties
+
+  Background: Open Experimental Date Input classic component page
+    Given I open "Experimental Date Input" component page classic
+
+  @positive
+  Scenario: Disable Date Input
+    When I disable DateInput component
+    Then Date input is disabled
+
+  @negative
+  Scenario: Disable and enable Date Input
+    When I disable DateInput component
+      And I enable DateInput component
+    Then Date input is enabled
+
+  @positive
+  Scenario: Date Input component is readOnly
+    When I check readOnly checkbox
+    Then Date input component is readOnly
+
+  @negative
+  Scenario: Date Input component is not readOnly
+    When I check readOnly checkbox
+      And I uncheck readOnly checkbox
+    Then Date input component is not readOnly
+
+  @positive
+  Scenario Outline: Change DateInput component field help to <fieldHelp>
+    When I set fieldHelp to "<fieldHelp>"
+    Then fieldHelp on preview is set to "<fieldHelp>"
+    Examples:
+      | fieldHelp               |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  # @ignore because of FE-1447
+  # | <>                       |
+
+  @positive
+  Scenario Outline: Change DateInput label to <label>
+    When I set label to "<label>"
+    Then label on preview is "<label>"
+    Examples:
+      | label                   |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  # @ignore because of FE-1447
+  # | <>                       |
+
+  @positive
+  Scenario: Enable label inline checkbox for Date Input component
+    When I set label to "labelSample"
+      And I check labelInline checkbox
+    Then label is set to inline
+
+  @positive
+  Scenario Outline: Change Date Input component label align to <labelAlign>
+    When I set label to "<label>"
```

Comment:
`"<label>"` no need to put this string in `<>` it's not a parameter

Reply:
fixed